webpack.config.js

```diff
@@ -103,7 +103,7 @@ const server = {
 };
 
 const client = {
-  entry: "./src/client/static/index.tsx",
+  entry: ['whatwg-fetch', "./src/client/static/index.tsx"],
```

Comment:
why do we need `whatwg-fetch`?

Reply:
https://github.com/guardian/gateway/pull/139#discussion_r551365369